distribution/docker/Dockerfile

```diff
@@ -17,8 +17,10 @@
 # under the License.
 #
 
-ARG JDK_VERSION=8
-FROM maven:3.8.4-jdk-11-slim as builder
+ARG JDK_VERSION=11
+
+FROM maven:3.8.6-jdk-11-slim as builder
```

Comment:
to make the build work on arm64 we could address the issue by forcing the platform to use amd64 using `FROM --platform=linux/amd64` (let's make sure we add a comment why and link to the node issue)

Reply:
It's a very good idea. 

But for Mac M1/M2, as I said above, it takes too long to build the distribution file at the build stage. I will also add some instructions in the doc to guide people to build the distribution out of docker.